.github/workflows/python-app.yml

```diff
@@ -36,6 +36,26 @@ jobs:
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
         
+          
+    - name: Test using unittest
+      run: |
+        python -m unittest
+      env:
+        TEST_TOKEN: ${{ secrets.TEST_TOKEN }}
+        DATABASE_URL: "${{ secrets.DATABASE_URI}}"
+        EMAIL: ${{ secrets.EMAIL }}
+        SECRET_KEY: ${{ secrets.SECRET_KEY }}
+
+    - name: Run Migrations
+      env:
+        TEST_TOKEN: ${{ secrets.TEST_TOKEN }}
+        DATABASE_URL: "postgresql://ypfrxmvtyuyudw:e0663560718f22f8b2fb44310d5da8f6e5b95f6be52476dc5f3e97316ec78ba6@ec2-18-210-159-154.compute-1.amazonaws.com:5432/dbkbegkmfukgcp"
```

Comment:
Why do you have your database URL here and not as an env var? 🤔

Reply:
I didn't have access to the settings page on the repo. Had to create a temporary db to test